File: java/Q14-Longest-Common-Prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String s="";
        for(int i=0;i<strs[0].length();i++){
            for(int j=0;j<strs.length;j++){
                if(i==strs[j].length() || strs[j].charAt(i)!=strs[0].charAt(i)){
                    return s;
                }
            }
            s+=strs[0].charAt(i);
        }
        return s;
    }
}

File: javascript/Q14-Longest-Common-Prefix.js
/**
 * @param {string[]} strs
 * @return {string}
 */
 var longestCommonPrefix = function(strs) {
    var len=strs[0].length;
    var res="";
    for(let i=0;i<len;i++){
        for(e of strs){
            if(i==e.length || e[i]!=strs[0][i]){
                return res;
            }
        }
        res+=strs[0][i];
    }
    return res;
};